README.md

```diff
@@ -1 +1,65 @@
-# SAMPL9
\ No newline at end of file
+# The SAMPL9 Blind Predicton Challenges for Computational Chemistry
+
+
+Challenge details, inputs, and results from the SAMPL8 series (phase) of challenges. Each individual SAMPL8 challenge may be broken up into multiple stages. 
+
+See the [SAMPL website](https://samplchallenges.github.io/) for information on the Statistical Assessement of the Modeling of Proteins and Ligands (SAMPL) series of challenges as a whole. This repository focuses specifically on the SAMPL9 sreies of challenges. Additionally, see the [SAMPL community](https://zenodo.org/communities/sampl/?page=1&size=20) on Zenodo for content related to the SAMPL series of challenges. If you wish to use Zenodo to post our presentaiton slides, datasets, and/or other SAMPL related material, and get a DOI for them, please upload to the community [here](https://zenodo.org/communities/sampl/?page=1&size=20) so that your content will be listed. 
+
+Because these files are available publicly, we have no record of who downloads them. Therefore, you should sign up for notifications. Specifically, if you want to receive updates if we uncover any problems, it is imperative that you either (a) [sign up for the SAMPL e-mail list](https://mailchi.mp/e36018629725/sampl9-sign-ups), or (b) sign up for notifications of changes to this GitHub repository (the Watch button, above); ideally you would do both. 
+
+Please note that some aspects of the SAMPL8 series of challenges are still ongoing, but as we are launching a new host-guest challenge that marks the beginning of the SAMPL9 series of challenges, so we have opened up this repository. 
+
+## Achnowleding and citing SAMPL
+
+If you've benefitted from our work on the SAMPL series of challenges, please be sure to acknowledge our SAMPL NIH grant in any publications/presentations. This funded host-guest experimetns, as well as our work organizing and administrating these challenges. You may acknowledge SAMPL by saying thing like, "We appreciate the National Institutes of Health for its support of the SAMPL project via R01GM124270 to David L. Mobley (UC Irvine)."
+
+We also ask you to cite the SAMPL dataset(s) you used. These are versioned on Zenodo, and the latest DOI is here: [![DOI](svg item goes here)](badge latestdoi will go here ). Click through for access to all data releases. You may cite these sets by their DOI. 
```

Comment:
I'd put in a placeholder like "will be linked here" for the initial merge, since we can't do a DOI after we cut a release, which can't happen until after we merge.